doc/release-notes.md

```diff
@@ -103,49 +46,68 @@ smart fee slider. It will only present an approximate fee calculated using the
 default target. The fee calculated using the correct target is still applied to
 the transaction and shown in the final send confirmation dialog.
 
-0.14.2 Change log
+0.14.3 Change log
 =================
 
 Detailed release notes follow. This overview includes changes that affect
 behavior, not code moves, refactors and string updates. For convenience in locating
 the code changes and accompanying discussion, both the pull request and
 git merge commit are mentioned.
 
+### Consensus
+ - #14247 `52965fb` Fix crash bug with duplicate inputs within a transaction (TheBlueMatt, sdaftuar)
+ 
 ### RPC and other APIs
-- #10410 `321419b` Fix importwallet edge case rescan bug (ryanofsky)
+
+- #10445 `87a21d5` Fix: make CCoinsViewDbCursor::Seek work for missing keys (Pieter Wuille, Gregory Maxwell)
+- #9853 Return correct error codes in setban(), fundrawtransaction(), removeprunedfunds(), bumpfee(), blockchain.cpp (John Newbery)
+
 
 ### P2P protocol and network code
-- #10424 `37a8fc5` Populate services in GetLocalAddress (morcos)
-- #10441 `9e3ad50` Only enforce expected services for half of outgoing connections (theuni)
+
+- #10234 `d289b56` [net] listbanned RPC and QT should show correct banned subnets (John Newbery)
 
 ### Build system
-- #10414 `ffb0c4b` miniupnpc 2.0.20170509 (fanquake)
-- #10228 `ae479bc` Regenerate bitcoin-config.h as necessary (theuni)
+
 
 ### Miscellaneous
-- #10245 `44a17f2` Minor fix in build documentation for FreeBSD 11 (shigeya)
-- #10215 `0aee4a1` Check interruptNet during dnsseed lookups (TheBlueMatt)
+
+ - #10451 `3612219` contrib/init/bitcoind.openrcconf: Don't disable wallet by default (Luke Dashjr)
+ - #10250 `e23cef0` Fix some empty vector references (Pieter Wuille)
+ - #10196 `d28d583` PrioritiseTransaction updates the mempool tx counter (Suhas Daftuar)
+ - #9497 `e207342` Fix CCheckQueue IsIdle (potential) race condition and remove dangerous constructors. (Jeremy Rubin)
 
 ### GUI
-- #10231 `1e936d7` Reduce a significant cs_main lock freeze (jonasschnelli)
+
+- #9481 `7abe7bb` Give fallback fee a reasonable indent (Luke Dashjr)
+- #9481 `3e4d7bf` Qt/Send: Figure a decent warning colour from theme (Luke Dashjr)
+- #9481 `e207342` Show more significant warning if we fall back to the default fee (Jonas Schnelli)
 
 ### Wallet
-- #10294 `1847642` Unset change position when there is no change (instagibbs)
+
+ - #10308 `28b8b8b` Securely erase potentially sensitive keys/values (tjps)
+ - #10265 `ff13f59` Make sure pindex is non-null before possibly referencing in LogPrintf call. (Karl-Johan Alm)
```

Comment:
Some PR has space in front of `-`. Is that intentional?

Reply:
Fixed.